sdk/cosmos/azure-cosmos/azure/cosmos/documents.py

```diff
@@ -173,6 +174,7 @@ class PartitionKind(object):
     """
 
     Hash = "Hash"
+    MultiHash = "MultiHash"
```

Comment:
When the customer calls create_container, and they pass the PK and Kind, can we validate that they cannot pass, for example, kind "Hash" with a PK with multiple components and viceversa?

Reply:
If the customer manually defines the kind as "Hash" while providing multiples components it will just result in a bad request with an error message telling them how many components they provided and how many were expected. In this case we would want create_container to result in an error.